src/common/valtext.cpp

```diff
@@ -332,6 +388,149 @@ void wxTextValidator::OnChar(wxKeyEvent& event)
     event.Skip(false);
 }
 
+// ----------------------------------------------------------------------------
+// wxTextValidator
+// ----------------------------------------------------------------------------
+
+wxIMPLEMENT_DYNAMIC_CLASS(wxTextValidator, wxTextValidatorBase);
```

Comment:
`Base` classes are excluded from wxRTTI hierarchy as "uninteresting", so `wxValidator` should be used instead.